logger.go

```
@@ -79,11 +83,58 @@ func setupTargetLogger(flags commandLineFlags, logTarget, commandOutput string)
+func (w logUploadingLogCloser) Close() error {
+	err := w.LogCloser.Close()
+	if err != nil {
+		return err
+	}
+	// Open logfile for reading
+	logData, err := os.Open(w.logfilePath)
+	if err != nil {
+		return err
+	}
+	// Upload logfile to server
+	req, err := http.NewRequestWithContext(context.TODO(), "POST", w.logUploadTarget, logData)
```

golangci-lint (1.64.8)

"POST" can be replaced by http.MethodPost

(usestdlibvars)